crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -1601,6 +1608,7 @@ impl<'db> TypeInferenceBuilder<'db> {
             ast::Expr::Await(await_expression) => self.infer_await_expression(await_expression),
             ast::Expr::IpyEscapeCommand(_) => todo!("Implement Ipy escape command support"),
         };
+        tracing::trace!("=> {}", ty.display(self.db));
```

Comment:
not sure we need this in the default tracing output

```suggestion
```

Reply:
That was temporary code that I missed removing